hack/sync-ingresscontroller-crd.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+API_CRD="vendor/github.com/openshift/api/operator/v1/0000_50_ingress-operator_00-custom-resource-definition.yaml"
+INGRESS_CRD="manifests/00-custom-resource-definition.yaml"
+COPY_CMD=$(cp $API_CRD $INGRESS_CRD)
```

Comment:
This executes `cp $API_CRD $INGRESS_CRD` and puts its output in `COPY_CMD`.  You probably want a function:
```shell
copy_crd() {
  cp "$API_CRD" "$INGRESS_CRD"
}
```
Then change `$COPY_CMD` to `copy_cmd` below.